aws/logs_monitoring/README.md

```diff
@@ -365,6 +365,10 @@ To test different patterns against your logs, turn on [debug logs](#troubleshoot
 
 `AdditionalTargetLambdaARNs`
 : Comma separated list of Lambda ARNs that will get called asynchronously with the same `event` the Datadog Forwarder receives.
+`InstallFromLayer`
```

Comment:
I would rename this to `InstallAsLayer`. `InstallFromLayer` makes it sound like we are somehow copying something off of a layer.

Reply:
We could also flip it and make this `InstallAsZip`? With a default of `false`?